### ISSUE TITLE
Logging API behavior for error differs

### ISSUE THREAD

**Issue by mgh87** [hirosassa/zerodriver#28]
The *.err(err Error) API from zerodriver is not behaving like the one from zerolog.

The execution of the following command
```
package main

import (
	"github.com/hirosassa/zerodriver"
	"github.com/rs/zerolog/log"
)

func main() {
	driverLog := *zerodriver.NewProductionLogger()
	log.Logger = *driverLog.Logger
	driverLog.Err(nil).Msg("error")
	log.Err(nil).Msg("error")
}
```
is
```
{"severity":"ERROR","time":"2022-12-02T16:16:39.642678263+01:00","message":"error"}
{"severity":"INFO","time":"2022-12-02T16:16:39.642704011+01:00","message":"error"}
```
This is the result of the usage of the wrong API in the abstraction layer at:
https://github.com/hirosassa/zerodriver/blob/e5d03f75d597c3ed055caa30e1c8e0ce12dff8e5/logger.go#L94
Which should be replaced with
```
e := l.Logger.Err(err)
```

**Comment by hirosassa** [comment]
This issue solved in #29

**Comment by hirosassa** [comment]
@mgh87 I released v0.1.4 including your contribution. Thanks again!
